core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -248,8 +248,19 @@ public long acceptWay(ReaderWay way) {
         if (!highwaySpeeds.containsKey(highwayValue))
             return 0;
 
+        String sacScale = way.getTag("sac_scale");
```

Comment:
There is already a test for this, which failed. I would not have recognized this otherwise!